packages/extension-browser/src/content-script/connector.ts

```diff
@@ -1,23 +1,23 @@
 import * as url from 'url';
 
 import { Engine } from 'hint';
+import { HttpHeaders } from '@hint/utils/dist/src/types/http-header';
 import { getContentTypeData, getType } from 'hint/dist/src/lib/utils/content-type';
 import {
     ConnectorOptionsConfig,
-    HttpHeaders,
     IConnector,
     FetchEnd,
     NetworkData,
     HTMLDocument,
     HTMLElement
 } from 'hint/dist/src/lib/types';
-import getElementByUrl from 'hint/dist/src/lib/utils/dom/get-element-by-url';
+import { getElementByUrl } from 'hint/dist/src/lib/utils/dom/get-element-by-url';
```

Comment:
I think I said this previously, we should move this to a utils or similar. Otherwise it looks like we are using an internal API.